x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_enrollment_flyout/index.tsx

```diff
@@ -129,12 +129,12 @@ export const AgentEnrollmentFlyout: React.FunctionComponent<Props> = ({
 
       <EuiFlyoutBody
         banner={
-          fleetServerHosts.length === 0 ? (
+          fleetServerHosts.length === 0 && mode === 'managed' ? (
```

Comment:
I also updated the Add agent flyout so that the `Run standalone` tab can be viewed without Fleet Server being setup. I thought this was appropriate since, well, those are standalone agents! @nchaulet maybe you can help me double check if this makes sense.

Reply:
Yes it make sense 👍